lib/smart_answer_flows/check-uk-visa/outcomes/outcome_no_visa_needed.govspeak.erb

```diff
@@ -1,10 +1,25 @@
 <% content_for :title do %>
-  You do not need a visa to come to the UK
+  <% if calculator.passport_country_ireland_or_outside_eea? %>
+    You do not need a visa to come to the UK
+  <% elsif calculator.passport_country_in_eea? %>
+    You do not need a visa to come to the UK before 31 December 2020
+  <% end %>
 <% end %>
 
 <% content_for :body do %>
 
-  ^When the UK leaves the EU, the rules for how long you can stay without a visa may change. [Check if you need to act now](/guidance/visiting-the-uk-after-brexit) so you can travel as planned.
+  <% if calculator.passport_country_is_ireland? %>
+    Your rights to live, work and study in the UK have not changed now that the UK has left the EU.
+  <% elsif calculator.passport_country_in_eea? %>
+    Your rights to live, work and study in the UK will not change until then.
```

Comment:
I might be reading this sentence out of context but is something missing here?

Reply:
It comes under the heading
` You do not need a visa to come to the UK before 31 December 2020`

Reply:
Ah okay then